### ISSUE TITLE
Please reopen 50417

### ISSUE THREAD

**Issue by rbtcollins** [microsoft/vscode#88337]
I believe it still happens per the bug I opened, misguidedly, on npm - https://github.com/nodejs/node/issues/31269

**Comment by connor4312** [comment]
Does this occur with all extensions disabled as well?

**Comment by rbtcollins** [comment]
Is there some easy way to save/restore the set of extensions to do such troubleshooting? If the answer is no - I'm in the middle of time critical work right now and unless this is super easy its probably going to be easier to answer that question yourself.

**Comment by connor4312** [comment]
Yep, you can launch code with `code --disable-extensions` to turn off extensions for just the current instance.

**Comment by rbtcollins** [comment]
Ah, so I can't test this with extensions disabled sorry. I attempted it:(. I'm working from a windows desktop (as explained in the node bug I filed) - without extensions the remote-wsl extension isn't present, and so I can't get into the context of where I would be able to reproduce the issue. Trying to install the remote extension after launching with `code --disable-extensions ` results in an error about the gallery missing.
